fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/QuerySegmentAggregator.java

```diff
@@ -103,10 +105,15 @@ protected void addQueryData(SqlQueryData querySegment,QueryParameter queryParm)
         //parallel arrays
         this.querySegments.add(querySegment);
         
-        String name = queryParm.getCode();
-        if("_id".compareTo(name)==0) {
-            queryParamId = queryParm;
-        } 
+        String code = queryParm.getCode();
+        if("_id".compareTo(code)==0) {
```

Comment:
minor style suggestion
```suggestion
        if ("_id".equals(code)) {
```

Reply:
next pr